### PR TITLE
Mantidplot/ Testing/ buildconfig/ misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -I ../mantid-word-whitelist.txt` where whitelist consists of  
```
alloced
allws
amin
ang
ans
arithmetics
dum
focussed
dur
iff
isnt
ith
lod
minimise
mut
mye
nd
originaly
realy
sav
splitted
te
tim
uint
vertexes
```

### CODE REVIEW

File: Testing/Tools/cxxtest/python/cxxtest/__init__.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 .. _CxxTest: http://cxxtest.tigris.org/
 
-The _CxxTest testing framework is focussed on being a lightweight
+The _CxxTest testing framework is focused on being a lightweight
```
Comment (rosswhitfield) [inline Testing/Tools/cxxtest/python/cxxtest/__init__.py]: Isn't **focussed** on the whitelist :confused:

**Review by rosswhitfield — CHANGES_REQUESTED** [review]

File: Testing/Tools/cxxtest/python/cxxtest/__init__.py (hunk 1 above)
Comment (luzpaz) [reply Testing/Tools/cxxtest/python/cxxtest/__init__.py]: Yea, its PEBKAC. I `git stash`ed some commits before I submitted the previous PRs and I keep `git stash pop`ing those commits and appending tthem to these current PRs when they are obsolete. So I need to just go back and clean out any obsolete issues like this one. Apologies.

Comment (luzpaz) [reply Testing/Tools/cxxtest/python/cxxtest/__init__.py]: fixed in 10d2a39

**Review by rosswhitfield — CHANGES_REQUESTED** [review]
You need to run clang-format v5 on your c++ changes, or just grab the [patch file](http://builds.mantidproject.org/job/pull_requests-clang-format/26646/artifact/0001-clang-format-PR23627-10d2a395.patch) from the build server and `git am` it. Thanks!

**Comment by luzpaz** [comment]
@rosswhitfield 
```
[foo@foo mantid]$ git am 0001-clang-format-PR23627-10d2a395.patch 
Applying: clang-format PR23627 10d2a395
error: patch failed: MantidPlot/src/AxesDialog.cpp:1512
error: MantidPlot/src/AxesDialog.cpp: patch does not apply
error: patch failed: MantidPlot/src/ConfigDialog.cpp:2436
error: MantidPlot/src/ConfigDialog.cpp: patch does not apply
error: patch failed: MantidPlot/src/CurvesDialog.cpp:476
error: MantidPlot/src/CurvesDialog.cpp: patch does not apply
error: patch failed: MantidPlot/src/DataPickerTool.h:59
error: MantidPlot/src/DataPickerTool.h: patch does not apply
error: patch failed: MantidPlot/src/GridDetails.h:68
error: MantidPlot/src/GridDetails.h: patch does not apply
error: patch failed: MantidPlot/src/ImportASCIIDialog.cpp:431
error: MantidPlot/src/ImportASCIIDialog.cpp: patch does not apply
error: patch failed: MantidPlot/src/LineProfileTool.h:82
error: MantidPlot/src/LineProfileTool.h: patch does not apply
error: patch failed: MantidPlot/src/MultiPeakFitTool.h:61
error: MantidPlot/src/MultiPeakFitTool.h: patch does not apply
error: patch failed: MantidPlot/src/RangeSelectorTool.h:88
error: MantidPlot/src/RangeSelectorTool.h: patch does not apply
error: patch failed: MantidPlot/src/ScreenPickerTool.h:56
error: MantidPlot/src/ScreenPickerTool.h: patch does not apply
error: patch failed: MantidPlot/src/ScriptingEnv.h:71
error: MantidPlot/src/ScriptingEnv.h: patch does not apply
error: patch failed: MantidPlot/src/TranslateCurveTool.h:66
error: MantidPlot/src/TranslateCurveTool.h: patch does not apply
Patch failed at 0001 clang-format PR23627 10d2a395
Use 'git am --show-current-patch' to see the failed patch
When you have resolved this problem, run "git am --continue".
If you prefer to skip this patch, run "git am --skip" instead.
To restore the original branch and stop patching, run "git am --abort".
```

**Comment by rosswhitfield** [comment]
Strange. But no problem, I did it for you :wink:

**Review by rosswhitfield — APPROVED** [review]

**Pull request merged**